packages/core/content-manager/admin/src/history/components/VersionInputRenderer.tsx

```diff
@@ -218,6 +218,44 @@ type VersionInputRendererProps = DistributiveOmit<EditFieldLayout, 'size'> & {
   shouldIgnoreRBAC?: boolean;
 };
 
+/**
+ * Checks if the i18n plugin added a label action to the field and modifies it
+ * to adapt the wording for the history page.
+ */
+const getLabelAction = (labelAction: VersionInputRendererProps['labelAction']) => {
+  if (!React.isValidElement(labelAction)) {
+    return labelAction;
+  }
+
+	// TODO: find a better way to do this rather than access internals
+  const labelActionTitleId = labelAction.props.title.id;
+
+  if (labelActionTitleId === 'i18n.Field.localized') {
+    return React.cloneElement(labelAction, {
+      ...labelAction.props,
+      title: {
+        id: 'history.content.localized',
+        defaultMessage:
+          'This value is specific to this locale. If you restore this version, the content will not replaced for other locales.',
```

Comment:
Looks like you might have a typo here:

```suggestion
          'This value is specific to this locale. If you restore this version, the content will not be replaced for other locales.',
```